cdap-ui/app/features/experimental/templates/directives-playground.html

```diff
@@ -32,7 +60,7 @@
   </div>
 </div>
 
-<div class="panel panel-default">
+<!-- <div class="panel panel-default">
```

Comment:
Commented? We are not using this in directives playground? If not can we remove?